common/config.go

```diff
@@ -25,6 +25,16 @@ type GridConfig struct {
 	SslVerify           string `toml:"ssl-verify" env:"SSL_VERIFY"`
 	HttpRequestTimeout  uint   `toml:"http-request-timeout" env:"HTTP_REQUEST_TIMEOUT"`
 	HttpPoolConnections uint   `toml:"http-pool-connections" env:"HTTP_POOL_CONNECTIONS"`
+	_wapiPassword       string
+}
+
+func (gridConfig *GridConfig) SetWapiPassword() {
+	gridConfig._wapiPassword = gridConfig.WapiPassword
+	gridConfig.WapiPassword = "******"
+}
```

Comment:
Instead of doing this you can implement the String() method to implement the Stringer interface which returns all fields except the Password.